Sluggo/View Controllers/TeamSelectorContainerViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
```

Comment:
Not sure what the intent of this container view is. It appears to wholly embed a navigation controller, in which case we could just subclass UINavigationController in such a way to enable injecting of identity. Let me know if I'm missing something here.

Reply:
(That said, this should be fine, it works for what we're doing. Navigation controller subclassing would be more effort, not necessarily needed)

Reply:
I'm using it to launch itself modally using  constructor injection. I can revisit this to see if there's a better way.